src/components/PublishingStatus.test.tsx

```diff
@@ -0,0 +1,18 @@
+import { screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import { setup, ALICE, renderApp, typeNewNode } from "../utils.test";
+
+test("Publishing Status is shown per relay", async () => {
+  const [alice] = setup([ALICE]);
+  const view = renderApp(alice());
+  await typeNewNode(view, "New Note");
+  const publishingStatusBtn = screen.getByLabelText("publishing status");
+  userEvent.click(publishingStatusBtn);
+  await screen.findByText("Publishing Status");
+  screen.getByText("Relay wss://relay.damus.io/:");
+  expect(
+    screen.getAllByText(
+      "100% of the last 3 events could be published on this relay"
+    )
+  ).toHaveLength(4);
+});
```

Comment:
More extensive tests are needed.

* Show that failing events are displayed
* Show that this stuff works on a per relay basis and can produce different results